src/pybricks/_common.py

```diff
@@ -370,116 +370,6 @@ def dc(self, duty):
         """
 
 
-class Speaker:
```

Comment:
This class should stay here and be modified to reflect what is implemented on SPIKE Prime.